test-cases/performance/perf-regression.100threads.100M-keys-z3-enterprise.yaml

```diff
@@ -0,0 +1,50 @@
+test_duration: 300
+
+stress_cmd_w: "cassandra-stress write no-warmup cl=QUORUM duration=60m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=200 -pop seq=1..100000000"
+prepare_write_cmd: "cassandra-stress write no-warmup cl=QUORUM n=100000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=200 -pop seq=1..100000000"
+stress_cmd_r: "cassandra-stress read no-warmup cl=QUORUM duration=60m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=800 -pop 'dist=gauss(1..100000000,50000000,5000000)' "
+stress_cmd_m: "cassandra-stress mixed no-warmup cl=QUORUM duration=60m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=300 -pop 'dist=gauss(1..100000000,50000000,5000000)' "
+stress_multiplier_w: 4
+stress_multiplier_r: 1
+stress_multiplier_m: 2
+
+# NOTE: following is needed for the K8S case
+k8s_loader_run_type: 'static'
+
+n_db_nodes: 3
+n_loaders: 6
+n_monitor_nodes: 1
+
+# AWS
+instance_type_db: 'i4i.2xlarge'
+instance_type_loader: 'c5.2xlarge'
+instance_type_monitor: 't3.small'
+#------
+
+#GCE
+use_preinstalled_scylla: false
```

Comment:
Why not use images ?

Reply:
It's actually because we don't have io_properties for this image and startup fails with:

 (DatabaseLogEvent Severity.ERROR) period_type=one-time event_id=efa470b8-ce5c-4284-9b53-b37be9fee77e: type=DATABASE_ERROR regex=(^ERROR|!\s*?ERR).*\[shard.*\] line_number=983 node=perf-regression-z3-ubuntu-db-node-59027ee0-0-1
18:54:13  < t:2024-01-09 16:54:13,119 f:file_logger.py  l:101  c:sdcm.sct_events.file_logger p:ERROR > 2024-01-09T16:54:12.991+00:00 perf-regression-z3-ubuntu-db-node-59027ee0-0-1      !ERR | scylla[10088]:  [shard  0] init - I/O Scheduler is not properly configured! This is a non-supported setup, and performance is expected to be unpredictably bad.
18:54:13  < t:2024-01-09 16:54:13,120 f:events_device.py l:184  c:sdcm.sct_events.events_device p:WARNING > Limit (DatabaseLogEvent Severity.CRITICAL) period_type=one-time event_id=7344da1d-38fe-4bd8-943c-23279c6d4d7f: type=RUNTIME_ERROR regex=std::runtime_error line_number=991 node=perf-regression-z3-ubuntu-db-node-59027ee0-0-1
18:54:13  < t:2024-01-09 16:54:13,120 f:events_device.py l:184  c:sdcm.sct_events.events_device p:WARNING > 2024-01-09T16:54:12.992+00:00 perf-regression-z3-ubuntu-db-node-59027ee0-0-1      !ERR | scylla[10088]:  [shard  0] init - Startup failed: std::runtime_error (Bad I/O Scheduler configuration) severity to Severity.ERROR as configured
18:54:13  < t:2024-01-09 16:54:13,121 f:file_logger.py  l:101  c:sdcm.sct_events.file_logger p:ERROR > 2024-01-09 16:54:13.077 <2024-01-09 16:54:12.992>: (DatabaseLogEvent Severity.ERROR) period_type=one-time event_id=7344da1d-38fe-4bd8-943c-23279c6d4d7f: type=RUNTIME_ERROR regex=std::runtime_error line_number=991 node=perf-regression-z3-ubuntu-db-node-59027ee0-0-1
18:54:13  < t:2024-01-09 16:54:13,121 f:file_logger.py  l:101  c:sdcm.sct_events.file_logger p:ERROR > 2024-01-09T16:54:12.992+00:00 perf-regression-z3-ubuntu-db-node-59027ee0-0-1      !ERR | scylla[10088]:  [shard  0] init - Startup failed: std::runtime_error (Bad I/O Scheduler configuration)

Reply:
SMI code doesn't know those instances

Can create io_properties once, and we have a new configuration option that can copy files into scylla.d folder